src/App.tsx

```diff
@@ -1,20 +1,23 @@
 import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import React from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { Layout } from './components/Layout';
+import { HomePageComponent } from './components/HomePageComponent';
+import { NotFoundPage } from './components/NotFoundPage';
 
 export const App = () => {
   return (
-    <div data-cy="app">
-      <Navbar />
-
-      <div className="section">
-        <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
-        </div>
-      </div>
-    </div>
+    <Layout>
+      <Routes>
+        <Route path="/" element={<HomePageComponent />} />
+        <Route path="/home" element={<Navigate to={'/'} />} />
+        <Route path="/people" element={<PeoplePage />}>
+          <Route path="/people/:slug" element={<PeoplePage />} />
+        </Route>
```

Comment:
The nested route inside `/people` is redundant since it uses the same component `PeoplePage`. If `PeoplePage` is supposed to handle both `/people` and `/people/:slug`, you might want to handle the logic for rendering different content based on the presence of `:slug` within the `PeoplePage` component itself. Consider removing the nested route or adjusting the component logic.